libs/ng-utils/project.json

```diff
@@ -9,7 +9,8 @@
       "executor": "@nrwl/angular:package",
       "outputs": ["dist/libs/ng-utils"],
       "options": {
-        "project": "libs/ng-utils/ng-package.json"
+        "project": "libs/ng-utils/ng-package.json",
+        "updateBuildableProjectDepsInPackageJson": false
```

Comment:
I found this fucker after an hour of searching....
https://github.com/nrwl/nx/issues/18175#issuecomment-1680565038